### ISSUE TITLE
Wallaby skip *.test.ts files in jest with auto config

### ISSUE THREAD

**Issue by Tom910** [wallabyjs/public#2443]
### Issue description or question

Does not work for .test.ts files
![image](https://user-images.githubusercontent.com/2185721/84295958-f76d6b80-ab53-11ea-91bc-9e2af79bf828.png)

works fine for .spec.ts files
![image](https://user-images.githubusercontent.com/2185721/84296041-208dfc00-ab54-11ea-98c7-afe76e83415e.png)

### Wallaby diagnostics report
```
{ editorVersion: 'WebStorm 2020.1.2',
  pluginVersion: '1.0.177',
  editorType: 'IntelliJ',
  osVersion: 'darwin 19.4.0',
  nodeVersion: 'v10.17.0',
  coreVersion: '1.0.907',
  config:
   { diagnostics: {},
     testFramework: { version: 'jest@24.8.0', configurator: 'jest@24.8.0', reporter: 'jest@24.8.0', starter: 'jest@24.8.0', autoDetected: true },
     filesWithCoverageCalculated: [ 'packages/**/*.{ts,tsx,js,jsx}', '!**/lib/**/*' ],
     globalSetup: false,
     micromatch: true,
     files:
      [ { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true },
        { pattern: 'dist/|build/|coverage/|/\\.cache/|/\\.idea/|/\\.vscode/|/\\.git/|/\\.gitlab/',
          regexp: /dist\/|build\/|coverage\/|\/\.cache\/|\/\.idea\/|\/\.vscode\/|\/\.git\/|\/\.gitlab\//,
          ignore: true,
          trigger: true,
          load: true },
        { pattern: 'node_modules/', regexp: /node_modules\//, ignore: true, trigger: true, load: true },
        { pattern: 'test', regexp: /test/, ignore: true, trigger: true, load: true },
        { pattern: 'examples', regexp: /examples/, ignore: true, trigger: true, load: true },
        { pattern: 'lib/', regexp: /lib\//, ignore: true, trigger: true, load: true },
        { pattern: '__integration__', regexp: /__integration__/, ignore: true, trigger: true, load: true },
        { pattern: '.cache', regexp: /.cache/, ignore: true, trigger: true, load: true },
        { pattern: '.npm', regexp: /.npm/, ignore: true, trigger: true, load: true },
        { pattern: '**/**', ignore: false, trigger: true, load: true, order: 1 },
        { pattern: '**/__tests__/**/*.[jt]s?(x)', ignore: true, trigger: true, load: true },
        { pattern: '**/?(*.)+(spec|test).[tj]s?(x)', ignore: true, trigger: true, load: true } ],
     tests:
      [ { pattern: '/node_modules/', regexp: /\/node_modules\//, ignore: true, trigger: true, load: true, test: true },
        { pattern: 'dist/|build/|coverage/|/\\.cache/|/\\.idea/|/\\.vscode/|/\\.git/|/\\.gitlab/',
          regexp: /dist\/|build\/|coverage\/|\/\.cache\/|\/\.idea\/|\/\.vscode\/|\/\.git\/|\/\.gitlab\//,
          ignore: true,
          trigger: true,
          load: true,
          test: true },
        { pattern: 'node_modules/', regexp: /node_modules\//, ignore: true, trigger: true, load: true, test: true },
        { pattern: 'test', regexp: /test/, ignore: true, trigger: true, load: true, test: true },
        { pattern: 'examples', regexp: /examples/, ignore: true, trigger: true, load: true, test: true },
        { pattern: 'lib/', regexp: /lib\//, ignore: true, trigger: true, load: true, test: true },
        { pattern: '__integration__', regexp: /__integration__/, ignore: true, trigger: true, load: true, test: true },
        { pattern: '.cache', regexp: /.cache/, ignore: true, trigger: true, load: true, test: true },
        { pattern: '.npm', regexp: /.npm/, ignore: true, trigger: true, load: true, test: true },
        { pattern: '**/__tests__/**/*.[jt]s?(x)', ignore: false, trigger: true, load: true, test: true, order: 2 },
        { pattern: '**/?(*.)+(spec|test).[tj]s?(x)', ignore: false, trigger: true, load: true, test: true, order: 3 } ],
     filesWithNoCoverageCalculated: [],
     runAllTestsInAffectedTestFile: false,
     updateNoMoreThanOneSnapshotPerTestFileRun: false,
     compilers: {},
     preprocessors: {},
     maxConsoleMessagesPerTest: 100,
     autoConsoleLog: true,
     delays: { run: 0, edit: 100, update: 0 },
     workers: { initial: 0, regular: 0, recycle: false },
     teardown: undefined,
     hints:
      { ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
        ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
        commentAutoLog: '?',
        testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' } },
     automaticTestFileSelection: true,
     runSelectedTestsOnly: false,
     extensions: {},
     env: { type: 'node', params: {}, runner: '<homeDir>/.nvm/versions/node/v10.17.0/bin/node', viewportSize: { width: 800, height: 600 }, options: { width: 800, height: 600 }, bundle: true },
     reportUnhandledPromises: true,
     slowTestThreshold: 75,
     lowCoverageThreshold: 80,
     loose: true,
     configCode: 'auto.detect#2113456370' },
```

**Comment by ArtemGovorov** [comment]
Hi @Tom910,

We could not reproduce the issue. We've set up this sample repo https://github.com/ArtemGovorov/wallaby-2443 that you may try:

<img width="946" alt="Screen Shot 2020-06-11 at 11 55 08 am" src="https://user-images.githubusercontent.com/979966/84336347-6beaed80-abda-11ea-97de-f3b0231a1a08.png">

Could you please check if your `di.test.ts` file maybe has Cyrillic `е` in its name instead of Latin `e`? In this case running `npx jest` from CLI in your repo will also ignore the file.

If it's not the case, and our sample repo is working for you, can you please share a stripped down version of your repo with just an empty di.test.ts file (and config files, such as jest config, package.json, tsconfig, etc. required to run it), so that we could reproduce the issue?

**Comment by Tom910** [comment]
@ArtemGovorov create reproduce repo - https://github.com/Tom910/wallaby-skip-test-files
main cause in 
```
testPathIgnorePatterns: [
        '<rootDir>/test',
    ]
```
this config enable skip all `*.test.*` files

**Comment by ArtemGovorov** [comment]
@Tom910 Thanks for providing the repo, we are investigating the issue. The workaround for now is this: 

```diff
testPathIgnorePatterns: [
-        '<rootDir>/test',
+        '<rootDir>/test/',
    ]
```

**Comment by ArtemGovorov** [comment]
The issue is fixed and the fix is published in [the latest core Wallaby core v1.0.908](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update).
